SmartSeq/SC_Plate_QC/SC_Plate_QC.wdl

```diff
@@ -216,3 +222,26 @@ task graphPlate{
         bootDiskSizeGb: boot_disk_space
     }
 }
+
+task gsutil_cp{
+	File plate_qc_metrics
+	String? target_google_bucket = "gs://fc-735a9d10-0cf6-4ae5-a203-5e5522bf5c3c/tableau_files"
+
+	command <<<
+		#Run gsutil cp and capture its exit status
+		gsutil cp ${plate_qc_metrics} ${target_google_bucket}
+		gsutil_exit_status=$?
+
+		# Check if gsutil cp was successful
+		if [[ $gsutil_exit_status -eq 0 ]]; then
```

Comment:
Double brackets?

Reply:
Double brackets are more flexible than single brackets. If I use single bracket, I need to add quote to my variable. 
```if ["$gsutil_exit_status" -eq 0 ];then```